clients/src/main/resources/common/message/GetTelemetrySubscriptionsResponse.json

```diff
@@ -0,0 +1,60 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+{
+  "apiKey": 71,
+  "type": "response",
+  "name": "GetTelemetrySubscriptionsResponse",
+  "validVersions": "0",
+  "flexibleVersions": "0+",
+  "fields": [
+    {
+      "name": "ThrottleTimeMs", "type": "int32", "versions": "0+",
+      "about": "The duration in milliseconds for which the request was throttled due to a quota violation, or zero if the request did not violate any quota."
+    },
+    {
+      "name": "ErrorCode", "type": "int16", "versions": "0+",
+      "about": "The error code, or 0 if there was no error."
+    },
+    {
+      "name": "ClientInstanceId", "type": "uuid", "versions": "0+",
+      "about": "Assigned client instance id if ClientInstanceId was 0 in the request, else 0."
+    },
+    {
+      "name": "SubscriptionId", "type": "int32", "versions": "0+",
+      "about": "Unique identifier for the current subscription set for this client instance."
+    },
+    {
+      "name": "AcceptedCompressionTypes", "type": "[]int8", "versions": "0+",
+      "about": "Compression types that broker accepts for the PushTelemetryRequest."
```

Comment:
I forget: where is the mapping from these integer IDs to the algorithms?

Reply:
`org.apache.kafka.common.record.CompressionType` holds the details.